uaa/src/test/java/org/cloudfoundry/identity/uaa/orchestrator/OrchestratorClaimAT.java

```diff
@@ -1,124 +1,280 @@
- *     subcomponent's license, as noted in the LICENSE file.
- *******************************************************************************/
 package org.cloudfoundry.identity.uaa.orchestrator;
 
+import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.ge.predix.iam.kubernetes.enums.ApiEnvironment;
+import com.ge.predix.iam.kubernetes.enums.ServiceName;
+import com.ge.predix.iam.kubernetes.exception.ServiceInstanceProviderException;
+import com.ge.predix.iam.kubernetes.model.Claim;
+import com.ge.predix.iam.kubernetes.model.uaa.UaaClaimSpec;
+import com.ge.predix.iam.kubernetes.model.uaa.UaaForProvider;
+import com.ge.predix.iam.kubernetes.model.uaa.UaaRequest;
+import com.ge.predix.iam.kubernetes.utils.OrchestratorUtil;
+import io.kubernetes.client.openapi.ApiException;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.commons.lang.StringUtils;
+import org.cloudfoundry.identity.uaa.integration.util.IntegrationTestUtils;
+import org.cloudfoundry.identity.uaa.zone.model.OrchestratorZoneResponse;
 import org.junit.Before;
-import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.security.oauth2.client.OAuth2RestTemplate;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.web.client.RestTemplate;
 
-import com.ge.predix.iam.kubernetes.enums.ServiceName;
-import com.ge.predix.iam.kubernetes.exception.ServiceInstanceProviderException;
-import com.ge.predix.iam.kubernetes.model.Claim;
-import com.ge.predix.iam.kubernetes.utils.AssumeRoleUtil;
-import com.ge.predix.iam.kubernetes.utils.OrchestratorUtil;
+import java.util.Random;
+import static org.junit.Assert.*;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(classes = OrchestratorIntegrationTestConfig.class)
+@Slf4j
 public class OrchestratorClaimAT {
+    protected static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
-    @Value("${kubernetes.config.secondary-namespace}")
-    private String namespace;
+    private static final Logger LOGGER = LoggerFactory.getLogger(OrchestratorClaimAT.class);
+
+    private static final String UAA_ADMIN_CLIENT_SECRET = "ZHVtbXlBZG1pblNlY3JldDEyMyMh";
+
+    private static final String UAA_CLAIM_NAME_PREFIX = "uaa-it-";
+
+    public static final String ZONES_URI = "/orchestrator/zones";
+
+    private static HttpHeaders headers;
+
+    private Random random = new Random();
+
+    private final String UAA_SUBDOMAIN = "rest-uaa-subdomain" + random.nextInt(1000);
+
+    private UaaRequest uaaRequest;
```

Comment:
i think this uaaRequest var is unused, remove it if not in use

Reply:
Done

Reply:
Done